rust/rsmgp-example/Cargo.lock

```diff
@@ -1,7 +1,5 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
-
```

Comment:
Don't know why this happened 😄 I can revert this if needed

Reply:
Yes, revert it, the reason is probably difference between Buda's and your version of Cargo